config/manager/params.env

```diff
@@ -0,0 +1 @@
+codeflare-operator-controller-image=quay.io/opendatahub/codeflare-operator:v1.0.1
```

Comment:
Do we have an agreement with the ODH operator team to change the name from `odh-codeflare-operator-controller-image`?

Reply:
Good catch I haven't informed anyone yet of this change. Will I just drop a message into the ODH DW channel?

Reply:
late for the party, but, if i understand this correctly, we are not going to keep CFO in the near future for ODH and RHOAI. 
so the changes done in upstream(as project-codeflare) is irrelevant to ODH and RHOAI.
which means, purely from project-codeflare perspective, these changes are not needed?